spec/filters/grok_spec.rb

```diff
@@ -1,5 +1,11 @@
 # encoding: utf-8
 require "logstash/devutils/rspec/spec_helper"
+
+# running the grok code outside a logstash release means
+# LOGSTASH_HOME will not be defined, so let's set it here
+# to Dir.pwd before requiring the grok filter
+LogStash::Environment::LOGSTASH_HOME = Dir.pwd unless LogStash::Environment.const_defined?(:LOGSTASH_HOME)
```

Comment:
`Dir.pwd` will work but I think 

    File.expand_path("../../../", __FILE__)

would be better and always point to the grok plugin home?